### ISSUE TITLE
runtime_memory parameter is ignored

### ISSUE THREAD

**Issue by LachlanStuart** [lithops-cloud/lithops#398]
The `runtime_memory` parameter passed to `executor.map`/`executor.call_async` seems to be ignored. I can see in the logs that it selects the correct runtime and ensures that it exists, but it actually tries to start a runtime with the amount of memory specified in the config. 

E.g. the following code tries to allocate 500MB, but it fails with `MemoryError` because it runs in a 128MB runtime (set by the executor config) instead of the 1024MB runtime requested.

```python
def allocate_500mb():
    b = bytearray(500 * 1024 * 1024)
    for i in range(len(b)):
        b[i] = 1  # Touch the memory to ensure it's actually allocated


executor = function_executor(config=lithops_config, runtime_memory=128)

fs = executor.call_async(allocate_500mb, (), runtime_memory=1024)
executor.get_result(fs)
```

Sometimes this also causes it to try to start non-existent runtimes, because `select_runtime` is called with the `runtime_memory` from `map`'s arguments, but then the invoker starts a runtime with the `runtime_memory` from the executor config. This is the issue I mentioned in #395 - I tried to run `executor.map(..., runtime_memory=2048)`, and it created a 2048MB runtime, but then tried to launch a non-existent 256MB runtime because 256 is the default config value.

I did some debugging, and the problem seems to be [here](https://github.com/lithops-cloud/lithops/blob/master/lithops/invokers.py#L261-L262) - `_invoke` gets the `runtime_memory` from the config, not from the job. I'm not sure what the correct behavior is though - `runtime_memory` isn't in the `job_description` dict that is passed to `_invoke`, and potentially  the remote_invoker logic should have different `runtime_memory` to the non-remote_invoker.

This occurs in Lithops 2.2.0 and 2.2.1-snapshot (revision 99b3b4)

**Comment by JosepSampe** [comment]
@LachlanStuart Good finding. Thanks

**Comment by JosepSampe** [comment]
I already have a fix. Will fully test it and then I will submit it

**Comment by JosepSampe** [comment]
@LachlanStuart Added a fix in #400 . ready to test

**Comment by LachlanStuart** [comment]
@JosepSampe I can confirm this has been fixed in the latest `master` code
